### DISCUSSION TITLE
Add support for resetting records of Bot players and consider increasing the maximum allowed file size for submission.

### DISCUSSION THREAD

**Discussion by Nitasurin** [smly/mjai.app#24] [Ideas]
1. If we resubmit the AI model, past game records may affect our evaluation of the new bot's level. It would be very helpful if a feature could be added to allow clearing game records and scores when resubmitting. (Perhaps it already supports this feature, as I have not resubmitted yet.)
2. I noticed that the current limit for submission is less than 500MB. Have you considered increasing this limit in the future?

**Comment by smly** [comment]
Thanks for your interest!

> 1. resubmit

With today's release, I have reset all previous game records and submissions. This is because I changed the third-party authentication method from Twitter to Github. This is not because of bot ratings.

I don't think clearing game records will help with bot evaluation. Even if a bot's rating is low, it may still play against stronger bots. And if it beats strong bots, the rating will jump up a lot. Also, I believe that the rating will converge to an appropriate rating value because a sufficiently large number of games will be played.
Let's observe the results first.

> 2. size limit

Which size is enough for you? We have just relaxed the size limit to 1GB.

**Comment by Nitasurin** [comment]
1. When a certain amount of past game records are accumulated, statistics other than rating (including Avg rank, win rate, etc.) will be updated slowly.
2. For the Bot I plan to submit, 500MB is already sufficient (if I use high-level compression). However, this may affect some developers who use larger models from joining AIJansou. The new 1GB limit is very helpful.

Thank you very much for taking the time to implement new AIJansou .

**Comment by smly** [comment]
> statistics other than rating will be updated slowly.

That is correct. Completely agree. 

I would like to improve this issue in the future. For example, displaying statistics for each bot program version and weekly batch number looks good. I would like to see if the win rate or the deal-in rate changes when the program is updated, wouldn't you?

> The new 1GB limit is very helpful

Thanks a lot for your comments! Ideas and suggestions are very motivating to me. :+1:
